keras_rs/src/layers/embedding/jax/test_utils.py

```diff
@@ -311,14 +308,14 @@ def _create_samples(
 
 def stack_shard_and_put_tables(
     table_specs: Nested[TableSpec],
-    tables: Nested[jax.Array],
+    tables: T,
```

Comment:
Can we make this more specific?  I'm guessing there are tests or something that use jax.Array, but now others that use embedding.EmbeddingVariables?